Test/Stats/test_stats.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+import pytest
+
+import netket as nk
+import netket.variational as vmc
+from netket.operator import local_values
+from netket.stats import statistics
+
+
+def _setup():
+    g = nk.graph.Hypercube(3, 2)
+    hi = nk.hilbert.Spin(g, 0.5)
+
+    ham = nk.operator.Heisenberg(hi)
+
+    ma = nk.machine.RbmSpinV2(hi, alpha=2)
+    ma.init_random_parameters()
+
+    return hi, ham, ma
+
+
+def _test_stats_mean_std(hi, ham, ma, batch_size):
+    sampler = nk.sampler.MetropolisLocalV2(ma, batch_size=batch_size)
```

Comment:
please sure MetropolisLocal(...) since v2 is going to be removed in my PR

Reply:
Good point. Then we should probably merge that one first, so that `MetropolisLocal` supports `batch_size` as argument.